api/src/main/java/org/openmrs/module/patientqueueing/mapper/PatientQueueMapper.java

```diff
@@ -0,0 +1,177 @@
+package org.openmrs.module.patientqueueing.mapper;
+
+import java.io.Serializable;
+
+public class PatientQueueMapper implements Serializable {
```

Comment:
What is this class used for?

Reply:
Its complicated to create a simple object out of an object that inherits the base openmrs object. So this will be used to map the patient queue parameters to either string or integers as they are being passed to the interface.